api/serverV2_test.go

```diff
@@ -0,0 +1,41 @@
+package api
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	tracesdk "go.opentelemetry.io/otel/sdk/trace"
+
+	"github.com/iotexproject/iotex-core/config"
+	"github.com/iotexproject/iotex-core/testutil"
+)
+
+func TestServerV2Start(t *testing.T) {
+	require := require.New(t)
+	cfg := newConfig(t)
+	config.SetEVMNetworkID(1)
+	svr, bfIndexFile, _ := createServerV2(cfg, false)
+	defer func() {
+		testutil.CleanupPath(t, bfIndexFile)
+	}()
+
+	err := svr.Start()
+	require.NoError(err)
+
+	err = testutil.WaitUntil(100*time.Millisecond, 3*time.Second, func() (bool, error) {
+		err = svr.Stop()
```

Comment:
don't use `svr.Stop() == nil` to assure the start of server

Reply:
this test  checks whether running start and stop  can make errors through default port 14014 and 15014.   but the /readiness is through another port,  and it checks wether the machine is ready or running health before iotex-server starts, then generate metrics through default port 8080.   so  i think  add this is meaningless. and you ?

Reply:
oh, two `server` got mixed up